frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -91,6 +90,9 @@ export type DashboardProps = {
   isNavigatingBackToDashboard: boolean;
   addCardOnLoad?: DashCardId;
   editingOnLoad?: string | string[] | boolean;
+  location: Location;
```

Comment:
These come from the type from `frontend/src/metabase/dashboard/hoc/types.ts` that I deleted